modules/cookbook/nav-cookbook.adoc

```diff
@@ -10,8 +10,8 @@ endif::[]
 
 
 * xref:cookbook-overview.adoc[Cookbook]
-** xref:recipe1.adoc[Recipe 1]
-** xref:receipe2.adoc[Recipe 2]
+** xref:recipe-onboarding.adoc[Recipe - Onboarding]
```

Comment:
I think short dashes in titles are a little bit Unamerican.  Maybe, we'd better use colons?

Reply:
I don't even know if we need to prefix each with 'recipe' actually, considering that's all that's really in this book.